docs/releases/6.0.txt

```diff
@@ -269,6 +269,15 @@ Miscellaneous
 Features deprecated in 6.0
 ==========================
 
+Email
+-----
+
+* Setting :setting:`ADMINS` or :setting:`MANAGERS` to a list of (name, address)
```

Comment:
We need to add a note in `docs/internals/deprecation.txt`

Reply:
Done